spec/access_policies/contact_info_access_policy_spec.rb

```diff
@@ -35,19 +35,4 @@
     end
   end
 
-  context 'confirm' do
-    it 'can be accessed by anyone' do
-      expect(OSU::AccessPolicy.action_allowed?(:confirm, anon,
-                                               contact_info)).to eq true
-      expect(OSU::AccessPolicy.action_allowed?(:confirm, app,
-                                               contact_info)).to eq true
-      expect(OSU::AccessPolicy.action_allowed?(:confirm, temp,
-                                               contact_info)).to eq true
-      expect(OSU::AccessPolicy.action_allowed?(:confirm, user,
-                                               contact_info)).to eq true
-      expect(OSU::AccessPolicy.action_allowed?(:confirm, admin,
-                                               contact_info)).to eq true
-    end
-  end
-
```

Comment:
Were these specs moved into the controller spec below? If not, should we have retained them?